wscript

```diff
@@ -69,52 +70,59 @@ def _setup_openmp(conf):
     else:
         conf.end_msg("OpenMP found.", 'GREEN')
 
+def _check_mac_osx_version(version):
+    '''
+     Native     Encoded
+    -------    --------
+     10.5.0 == 0x0A0500
+      ^ ^ ^       ^ ^ ^
+      | | |       | | |
+      major       major
+        | |         | |
+        minor       minor
+          |           |
+          patch       patch
+
+    version -> 24-bit encoded version
+    '''
+
+    assert isinstance(version, int)
+    floor_version = version
+    osx_version = 0
+
+    floor_version_major = (floor_version >> 8 & 0xff)
```

Comment:
just to reiterate - me approving of the change doesn't mean we can't discuss it's value, I believe in everyone working together as a team and no one person dominating or mandating the conversation. If everyone thinks this section is better kept as human readable as possible we can revert to the old way.  I agree that another line of documentation would be good to add if we keep the current function.